README.md

```diff
@@ -66,7 +67,7 @@ A valid (read: that follows this interface) IPFS core implementation, must expos
 
 `callback` must follow `function (err, node) {}` signature, where `err` is an error if the operation was not successful and `node` is a MerkleDAG node of the type [DAGNode][]
 
-If no `callback` is passed, a promise is returned.
+Returns a [promise][].
```

Comment:
Interesting. Not sure if we want this though, I guess we have to fork it and make our own version in the long run.